tools/nl/embeddings/input/base/sheets_svs.csv

```diff
@@ -2760,6 +2764,10 @@ Count_ThunderstormWindEvent,Number of thunderstorm wind events
 Count_TornadoEvent,Number of tornado events
 Count_TropicalStormEvent,Number of tropical storm events
 Count_UnemploymentInsuranceClaim_StateUnemploymentInsurance,Number of state unemployment insurance claims
+Count_Vehicle_InvolvedInCrash_InTransport,Number of vehicles crashed in transport 
+Count_VehicleCrashIncident_NationalHighway,Number of crash incidents occurred on the national highway
+Count_VehicleCrashIncident_StateHighway,Number of crash incidents on state highway
+Count_VehicleCrashIncident_USHighway,Number of crash incidents on US highway

```

Comment:
US highway is a bit vague in the meaning and maybe very hard to match actual query. Looking at these 3 stat vars, I wonder if there are aggregate stat var?

For example, the most likely query is "how many highway car crashes happened in 2020", which stat vars do we want to match here?

Reply:
The raw dataset we processed is at a crash granularity which we aggregated. Looking at the data the USHighway or StateHighway may or may not be part of the National Highway Network. The US highway seems to be an older system as per https://en.wikipedia.org/wiki/Numbered_highways_in_the_United_States. Shall we remove the USHighway from the index and retain both National highway and State Highway.  Is the question "how many highway car crashes happened in 2020" returning both NationalHighway, StateHighway numbers a desirable behavior ?